packages/components/media-player/src/PlayerControls/TimeBox.js

```diff
@@ -1,20 +1,17 @@
 import React from 'react';
-import style from './index.module.css';
 
-class TimeBox extends React.Component {
+import style from './index.module.scss';
 
-  shouldComponentUpdate(nextProps) {
-    if (nextProps !== this.props) {
-      return true;
-    }
+class TimeBox extends React.Component {
 
-    return false;
+  shouldComponentUpdate = (nextProps) => {
+    return nextProps !== this.props;
```

Comment:
I think checking for object equality here will always return `true` (since it only checks that they're the same reference in memory). Better to compare on the relevant props that might have changed between renders.

Reply:
Hmm. Yeah this was to improve the rendering performance but without cluttering the code too much. Is there a working one-liner for this?

Reply:
Thinking either the lodash `isEqual` or perhaps this:
https://www.npmjs.com/package/react-fast-compare

have you used it before?